tests/integration/golang/fixtures/run.go

```diff
@@ -45,3 +47,51 @@ func (f RunFixtures) CreateTestRun(
 	}
 	return run, nil
 }
+
+// CreateTestRuns creates some num runs belonging to the experiment
+func (f RunFixtures) CreateTestRuns(
+	ctx context.Context, exp *models.Experiment, num int,
+) ([]*models.Run, error) {
+	var runs []*models.Run
+	// create runs for the experiment
+	for i := 0; i < num; i++ {
+		run := &models.Run{
+			ID:             strings.ReplaceAll(uuid.New().String(), "-", ""),
+			ExperimentID:   *exp.ID,
+			SourceType:     "JOB",
+			LifecycleStage: models.LifecycleStageActive,
+			Status:         models.StatusRunning,
+		}
+		run, err := f.CreateTestRun(context.Background(), run)
+		if err != nil {
+			return nil, err
+		}
+		runs = append(runs, run)
+	}
+	return runs, nil
+}
+
+// GetTestRuns fetches all runs for an experiment
+func (f RunFixtures) GetTestRuns(
+	ctx context.Context, experimentID int32) ([]models.Run, error) {
+	return f.runRepository.List(ctx, experimentID)
+}
+
+// FindMinMaxRowNums finds min and max rownum for an experiment's runs
+func (f RunFixtures) FindMinMaxRowNums(
+	ctx context.Context, experimentID int32) (int64, int64, error) {
+	runs, err := f.runRepository.List(ctx, experimentID)
```

Comment:
Do we need `list` only for fixtures?

Reply:
as I can se - yes. If so then please remove it. we decided today to keep repo layer only related to business logic. everything else you can add to `fixtures`.
cc: @jgiannuzzi

Reply:
Removed functionality to fixtures/run.go